README.md

```diff
@@ -10,6 +10,17 @@ This is a simple react app to manage your ETH2 validator keystores. It allows to
 
 [See more screenshots](images)
 
+## Prerequisites
+
+This tool requires nodejs 16 or better and yarn as well as react-scripts and typescript. Installation of nodejs and npm differs by OS, for Ubuntu 20.04: 
+
+`sudo apt update && sudo apt install -y curl` and then `curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -` and finally `sudo apt install -y nodejs`.
+
+Once nodejs and npm have been installed, install yarn: `sudo corepack enable` and test it's there with `yarn --version`
+> If this fails, install yarn via npm instead: `sudo npm install --global yarn`
+
+Install react-scripts and typescript: `yarn add react-scripts typescript`
```

Comment:
i dont think this is needed?

though it might need a `yarn install` before doing `yarn run` - that might be what you were missing?